tests/test_processes.py

```diff
@@ -350,7 +351,15 @@ def test_GasReinjectionCompressor(test_model):
     total = proc.energy.data.sum()
     expected = ureg.Quantity(64913.96925930, "mmbtu/day")
     assert approx_equal(total, expected, rel=10e-3)
-
+    
+def test_TransmissionCompressor(test_model):
+    analysis = test_model.get_analysis('test_gas_processes')
+    field = analysis.get_field('test_TransmissionCompressor')
+    field.run(analysis)
+    proc = field.find_process("TransmissionCompressor")
+    total = proc.energy.data.sum()
+    expected = ureg.Quantity(1733.3619, "mmbtu/day")
+    assert approx_equal(total, expected, rel=10e-3)
```

Comment:
The expected quantity here is copied from the return of `proc.energy.data.sum()`. I have no idea if these values are appropriate.